### PR TITLE
Add JNI fuse lib to the tarball

### PR DESCRIPTION
**Pull request by jiacheliu3**
The JNI fuse lib should be added to the tarball, so that users can build the fuse docker image

### CODE REVIEW

File: dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go [hunk 1]
```diff
@@ -204,11 +204,13 @@ func addAdditionalFiles(srcPath, dstPath string, hadoopVersion version, version
 		"conf/masters",
 		"conf/metrics.properties.template",
 		"conf/workers",
+		"integration/docker/.dockerignore",
```
Comment (jiacheliu3) [inline dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go:207]: I think this is useful too.

File: dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go [hunk 2]
```diff
@@ -204,11 +204,13 @@ func addAdditionalFiles(srcPath, dstPath string, hadoopVersion version, version
...
+		"integration/docker/conf/alluxio-env.sh.template",
+		"integration/docker/conf/alluxio-site.properties.template",
 		"integration/docker/Dockerfile",
 		"integration/docker/Dockerfile.fuse",
 		"integration/docker/entrypoint.sh",
-		"integration/docker/conf/alluxio-site.properties.template",
-		"integration/docker/conf/alluxio-env.sh.template",
```
Comment (jiacheliu3) [inline dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go:211]: Reorder according to alphabetic order

**Review by apc999 — APPROVED** [review]

**Comment by apc999** [comment]
Failing test "PinCommandIntegrationTest" has nothing to do with this PR.
Thanks for fix the issue @jiacheliu3 

LGTM

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**